dbt_project.yml

```diff
@@ -56,8 +408,22 @@ seeds:
       +schema: vocab_seeds
       +docs:
         node_color: '#69AED5'
-    synthea:
-      +enabled: true
-      +schema: synthea_seeds    
-      +docs:
-        node_color: '#69AED5'
+      concept_ancestor_seed: 
```

Comment:
I had to do this more verbose reference (vs line 429 for synthea seeds) because we need to give the vocab seeds different names than the final OMOP models.